.github/workflows/lint.yml

```diff
@@ -31,13 +31,14 @@ jobs:
         if: env.GIT_DIFF
         with:
           go-version-file: 'go.mod'
-      - uses: golangci/golangci-lint-action@v6
+      - uses: golangci/golangci-lint-action@v8
         if: env.GIT_DIFF
         with:
           # If you change this version, be sure to also change it in contrib/devtools/Makefile.
-          version: v1.60
-          args: --timeout 10m --out-${NO_FUTURE}format colored-line-number
+          version: v2.1.6
+          args: --timeout 10m --output.text.path=stdout --output.text.colors=true
           github-token: ${{ secrets.github_token }}
+          only-new-issues: true
```

Comment:
We shouldn't do this. From our terminals, we should be able to run `make lint` without it complaining about anything. With this `only-new-issues: true`, any existing problems won't be identified by the github action, but `make lint` will fail locally.

Usually, when we bump `golangcli-lint`, we get a new slew of failures that all need to be fixed (or get a `nolint` comment). Some of them can be fixed automatically using the `--fix` flag when invoking `golangcli-lint`. The `make lint-fix` target will do this too. It probably won't be able to fix everything, but it'll get some of it.